sentry_sdk/integrations/__init__.py

```diff
@@ -10,6 +10,8 @@
     from typing import Dict
     from typing import Iterator
     from typing import List
+    from typing import Optional
+    from typing import Sequence
```

Comment:
Please import `Sequence` from `collections.abc`. `typing.Sequence` is a [deprecated alias](https://docs.python.org/3/library/typing.html#typing.Sequence) to `collections.abc.Sequence`